### ISSUE TITLE
Additional preference: on login, automatically open workspace on file

### ISSUE THREAD

**Issue by LisaAlmarode** [ericwinger/Jade#585]
I would like to be able to automatically open workspaces from files, on login. 

My original thought was how annoying it is that I can't keep the console workspace contents alive from session to session.  But opening a separate workspace is probably more useful.

**Comment by ericwinger** [comment]
Should be able to open a workspace on a file or not in 3.0.82.

**Comment by LisaAlmarode** [comment]
I can't get this to work in 3.0.82.

This is my prefs file:
```
autocommitEnabled: false
autocompleteEnabled: false
transcriptEnabled: false

openWorkspace: C:\GS\Oscar\testws.gs
openBrowserOnPackage: RowanSample1-Tests
```
the file exists:

![image](https://user-images.githubusercontent.com/13142166/67437247-4191e480-f5a5-11e9-9ecc-5a86922dc7d6.png)

But on login, it opens the PB, but not the workspace.

**Comment by ericwinger** [comment]
Very odd. Full paths work for me. There are no spaces in your path (which I realize probably won't work as currently implemented).  If you move ``testws.gs`` to the working directory (where the .exe is) does it open?  Does a ``nil`` open a blank workspace ok?

**Comment by LisaAlmarode** [comment]
This file is in the same dir as the exe, so presumably working directory; and no, nil does not open a blank workspace either.

**Comment by ericwinger** [comment]
Very odd.  Are you sure that the preferences file that opens the workspace is called ``Jadeite.prefs`` and is in the same directory as the .exe?

**Comment by LisaAlmarode** [comment]
Same file it's always been - and it does opens the Project Browser, and set other options. You can look at it in person tomorrow...

**Comment by ericwinger** [comment]
Think the problem is that the ``openWorkspace:`` method in ``JadeitePreferences`` was getting stripped out of the runtime. Fixed in 3.0.83.

**Comment by LisaAlmarode** [comment]
working nicely in 3.0.83
